app/assets/javascripts/discourse/app/models/topic-tracking-state.js

```diff
@@ -83,20 +83,50 @@ const TopicTrackingState = EmberObject.extend({
    *
    * @method establishChannels
    */
-  establishChannels() {
-    this.messageBus.subscribe("/latest", this._processChannelPayload);
+  establishChannels(meta) {
+    this.messageBus.subscribe(
+      "/latest",
+      this._processChannelPayload,
+      meta ? meta["/latest"] : -1
```

Comment:
Kind of nitpicky but unless you have read the MB docs, `-1` is a bit magic, maybe we should put this in a const like `MESSAGE_BUS_ALL_NEW` or something. Just a suggestion, I am mainly concerned about people questioning what this means or changing it to other values which can do different things

Reply:
Actually, I had to do this just for the test enviornment. Let me see if we already have an existing way to preload things in our acceptance test.